cpp/ql/src/semmle/code/cpp/exprs/Expr.qll

```diff
@@ -964,6 +964,65 @@ class NoExceptExpr extends Expr, @noexceptexpr {
   }
 }
 
+/**
+ * A C++17 fold expression.
```

Comment:
Please clarify that this expression only appears in templates, not in their instantiations.

Reply:
Done.